redaxo/src/core/assets/standard.js

```diff
@@ -466,6 +466,50 @@ jQuery(function($){
     }
 });
 
+// save & restore sidebar panel toggle status via localstorage
+jQuery(function($){
+    var $sidebar = $('#rex-js-main-sidebar'),
+        sidebar_status,
+        $sections;
+    function store_sidebar_status () {
+        sidebar_status = [];
+        $sections.each(function (k, v) {
+            sidebar_status.push($(v).find('div.collapse').hasClass('in').toString());
```

Comment:
ich würde nicht nur `true/false` als simples Array in den storage packen. Wenn ein Addon die sidebar auch verwendet, würde man hier für beide immer das gleiche haben. 

(Ich habe mal eben gesucht, aber weder im Core noch bei FOR ein Beispiel gefunden.)